I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 315, in script
    jump therma_ext
ScriptError: could not find label 'therma_ext'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 315, in script
    jump therma_ext
  File "/Applications/renpy-8.0.1-sdk/renpy/ast.py", line 1974, in execute
    rv = renpy.game.script.lookup(target)
  File "/Applications/renpy-8.0.1-sdk/renpy/script.py", line 927, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'therma_ext'.

macOS-10.16-x86_64-i386-64bit x86_64
Ren'Py 8.0.1.22070801
Fantasy Ren'py GUI template 1.0
Mon Aug  8 11:25:16 2022
